src/core/server/saved_objects/migrationsv2/integration_tests/migration_7_13_0_transform_failures.test.ts

```diff
@@ -122,11 +140,11 @@ describe('migration v2', () => {
       }
 
       expect(errorMessage.includes('7 transformation errors were encountered:')).toBeTruthy();
-      expect(
-        errorMessage.includes(
-          'space:default: Error: Document "default" has property "space" which belongs to a more recent version of Kibana [6.6.0]. The last known version is [undefined]'
+      expect(errorMessage).toEqual(
+        expect.stringContaining(
+          'space:sixth: Error: Migration function for version 7.14.0 threw an error'
```

Comment:
I know we're only asserting agains the error message here but it would be great if we could also assert that the original stack trace is available too.

Reply:
We're already doing that in a dedicated test suite (`/type_migration_failure.test.ts`), and such assertions are quite tedious, which is why I thought it wasn't necessary to do it here too.

https://github.com/elastic/kibana/blob/05616481564986d7f18cfbcefeba93887a9d04fe/src/core/server/saved_objects/migrationsv2/integration_tests/type_migration_failure.test.ts#L133-L136